add tooltips for query types (#136)
VictoriaMetrics/victorialogs-datasource: CHANGELOG.md

## tip

> 

Reviewer: ```suggestion
```

> * FEATURE: add tooltips for query types and a warning to include `| stats` pipe.

Reviewer: ```suggestion
* FEATURE: add tooltips and info messages for query types. Now, plugin will warn about correct usage of `stats` panels and will provide more info about different query types.
```

* BUGFIX: fix bug with incomplete rendering of time series panels when selecting bigger time intervals.

> 

Reviewer: ```suggestion
```
